src/Target.php

```diff
@@ -87,35 +84,36 @@ abstract class Target extends Component
      *
      * The signature of the callable should be `function ($message)`.
      */
-    public $prefix;
+    private $prefix;
+
     /**
      * @var int how many messages should be accumulated before they are exported.
      * Defaults to 1000. Note that messages will always be exported when the application terminates.
      * Set this property to be 0 if you don't want to export messages until the application terminates.
      */
-    public $exportInterval = 1000;
+    private $exportInterval = 1000;
     /**
      * @var array the messages that are retrieved from the logger so far by this log target.
      * Please refer to [[Logger::messages]] for the details about the message structure.
      */
-    public $messages = [];
+    private $messages = [];
     /**
-     * @var bool whether to log time with microseconds.
-     * Defaults to false.
+     * @var string The date format for the log timestamp.
+     * Defaults to Y-m-d H:i:s.u
      */
-    public $microtime = false;
+    private $timestampFormat = 'Y-m-d H:i:s.u';
```

Comment:
Looks better than it was.

See https://github.com/yiisoft/log/issues/22 as well.